src/support/azext_support/_help.py

```diff
@@ -0,0 +1,204 @@
+      - name: Show problem classification 'VM Performance / Memory usage is higher than expected' for service 'Virtual Machine Running Window'.
+        text: >
+               az support services problem-classifications show \\
+                 --service-name "6f16735c-b0ae-b275-ad3a-03479cfa1396" \\
+                 --problem-classification-name "0cb121be-61df-48e4-5faa-b34f01c7aa16"
+"""
+
+helps['support tickets'] = """
+    type: group
+    short-summary: Commands to manage support tickets.
+"""
+
+helps['support tickets list'] = """
+    type: command
+    short-summary: List support tickets.
+    examples:
+      - name: List support tickets that are active and created after specific date".
+        text: |-
+               az support tickets list --filters "CreatedDate ge 2019-12-01 and Status eq 'Open'"
+"""
+
+helps['support tickets show'] = """
+    type: command
+    short-summary: Show support ticket.
+    examples:
+      - name: Show support ticket named "test_ticket_from_pythoncli".
+        text: |-
+               az support tickets show --ticket-name "test_ticket_from_pythoncli"
+"""
+
+helps['support tickets update'] = """
+    type: command
+    short-summary: Update support ticket.
+    examples:
+      - name: Update severity for support ticket named "test_ticket_from_pythoncli".
+        text: |-
+               az support tickets update --ticket-name "test_ticket_from_pythoncli" --severity "moderate"
+
+      - name: Update one or more properties of contact details for support ticket named "test_ticket_from_pythoncli".
+        text: >
+               az support tickets update --ticket-name "test_ticket_from_pythoncli" \\
+                --contact-additional-emails "xyz@contoso.com" "devs@contoso.com" \\
+                --contact-country "USA" \\
+                --contact-email "abc@contoso.com" \\
+                --contact-first-name "Foo" \\
+                --contact-language "en-US" \\
+                --contact-last-name "Bar" \\
+                --contact-method "phone" \\
+                --contact-phone-numbe "123-456-7890" \\
+                --contact-timezone "Pacific Standard Time"
+"""
+
+helps['support tickets create'] = """
+    type: command
+    short-summary: Create support ticket.
+    examples:
+      - name: Create support ticket for technical issue.
+        text: >
+               az support tickets create \\
+                --contact-country "USA" \\
+                --contact-email "abc@contoso.com" \\
+                --contact-first-name "Foo" \\
+                --contact-language "en-US" \\
+                --contact-last-name "Bar" \\
+                --contact-method "email" \\
+                --contact-timezone "Pacific Standard Time" \\
+                --description "test ticket description" \\
+                --problem-classification "/providers/Microsoft.Support/services/cddd3eb5-1830-b494-44fd-782f691479dc/problemClassifications/ffe93180-2ece-07c9-0cd4-6ae7ccc19d7c" \\
```

Comment:
Same here, a dummy value name like `MySupportProblemClassificationID` would be appreciated.

Reply:
Addressed in latest commit.